qa/automation/CareerPortal.Tests.Scenarios/Sataeva_cv form_feature/CV_form.feature

```diff
@@ -0,0 +1,181 @@
+Feature: Send CV form
+
+Background:
+Given user opened https://career.quantori.com/
+And user scrolled down to the "Send us your CV" form
+
+Scenario Outline: checking if the field "Name" is filled out with correct amount of symbols 
+When user entered text with <Amount> of symbols in "Name" field
+And "Name" field consists of 2 words
+And these two words are separated with one space
+Then the result should be <Result> on the screen
+
+Examples:
+  | Amount | Result                                                                                  | 
+  | 69     | no error message, line under the field is green                                         |
+  | 70     | no error message, line under the field is green                                         | 
+  | 71     | line under the field is red, error message "The maximum number of characters is 70"     | 
+
+
+Scenario: field "Name" consists of one word 
+When user types "Hello" in field "Name"
+Then user should get an error message "Invalid name format"
+And line under the field should turn to red
```

Comment:
We can use quotation marks ("") if we know that this text message might change its colour: for example - "red" colour or "green" colour.

Reply:
I did not really get what do you mean here. :(
Should I rewrite it somehow in this way: "And line under the field should change color to "red"?

Reply:
yes, just add quotation marks

Reply:
Will do it now.